content/docs/networking/ipv4-address-space.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "IPv4 Address Partitioning"
+date: 2023-04-21T12:56:27+02:00
+draft: false
+hidden: false
+---
+
+# IPv4 Address Partitioning
+
+In this page we go over the IPv4 address space partitioning for Oakestra.
+
+## Network Split
+
+Oakestra makes use of the reserved private IPv4 subnet `10.0.0.0/8` for its service addressing.
+
+The current implementation of Oakestra (Accordion) uses the following network split for IPv4 networking:
```

Comment:
We don't want to refer to any particular version here as later updates in code requires unnecessary text changes